packages/esm-service-queues-app/src/routes.json

```diff
@@ -70,6 +70,21 @@
       "name": "add-queue-entry-widget",
       "component": "addQueueEntry",
       "slot": "add-queue-entry-slot"
-    }
```

Comment:
These changes should be removed.  We are not adding these extensions any longer.